sdk/python/feast/driver_test_data.py

```diff
@@ -2,10 +2,10 @@
 import itertools
 from datetime import timedelta, timezone
 from enum import Enum
-from zoneinfo import ZoneInfo
```

Comment:
let's revert any python changes like this. these appear to be unrelated to this change.